config.yml

```diff
@@ -0,0 +1,8 @@
+telemetryaddr:
+metricspath:
+unifiaddr:
+username:
+password:
+site:
+insecure:
+timeout:
```

Comment:
I would call this file `config.yml.example` and put the default values (see above) or some example values in it, so people can see how the options are meant to be specified.